src/Components/MovieDetails/MovieDetails.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react'
 import './MovieDetails.css'
+import { NavLink } from 'react-router-dom'
 
 const MovieDetails = ({ singleMovie, displayHome }) => {
     return (
         <div className='movieDetailContainer'>
-            <button onClick={() => displayHome()} className ='homeButton'>Home</button>
+            <NavLink to='/'><button onClick={() => displayHome()} className ='homeButton'>Home</button></NavLink>
             <div className='movieDetails'>
                 <img className='poster-mini' src={singleMovie.poster_path} />
                 <div className='movieInfo'>
                     <h2>{singleMovie.title}</h2>
-                    {singleMovie.tagline.length>0 && <p>"{singleMovie.tagline}"</p>}
-                    <p>Genre: {singleMovie.genres.join(", ")}</p>
-                    <p>Rating: {singleMovie.average_rating.toFixed(2)}/10</p>
+                    {singleMovie.tagline && <p>"{singleMovie.tagline}"</p>}
```

Comment:
Great job using a conditional for each element to determine if they exists before rendering them since some movies don't have all info (rating, tagline, budget, etc.)